content/en/docs/tasks/configure-pod-container/user-namespaces.md

```diff
@@ -72,10 +72,10 @@ to `false`. For example:
    kubectl apply -f https://k8s.io/examples/pods/user-namespaces-stateless.yaml
    ```
 
-1. Attach to the container and run `readlink /proc/self/ns/user`:
+1. Add a debugging container and attach to it and run `readlink /proc/self/ns/user`:
 
    ```shell
-   kubectl attach -it userns bash
+   kubectl debug userns -it --image=busybox
    ```
```

Comment:
Why is this the right change to make? I think attaching to or exec-ing into the existing container was the intent here.

Reply:
Yes, as I mentioned in the [issue](https://github.com/kubernetes/website/issues/48675) either `exec` or `debug` are fine, I just chose `debug`

Reply:
I ran through on a local cluster and the debug command works as intended.